### PR TITLE
Create Tangle Dapp - EVM Staking App

### LINKED ISSUE #1677: [TASK] Create "app.tangle.tools" directory in webb-dapp (tangle-app)
# Overview
Create the scaffold for the tangle app where we will put the EVM staking app.

Tasks 
- [x] Create the directory and hook it up.

### LINKED ISSUE #1776: [TASK] Add sidebar nav to the tangle-dapp.

### PR DESCRIPTION
**Pull request by devpavan04**
## Summary of changes

- Creates a new NextJS app called `tangle-dapp` for `EVM Staking App`

### Reference issue to close (if applicable)

- Closes #1677 
- Closes #1776

### CODE REVIEW

File: apps/tangle-dapp/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  twitter: {
+    card: 'summary_large_image',
+    title: 'Tangle Dapp',
+    description: 'Welcome to Tangle Dapp!',
+  },
+};
+
+// revalidate every 5 seconds
+export const revalidate = 5;
```
Comment (AtelyPham) [inline apps/tangle-dapp/app/layout.tsx:42]: Please consider removing this, as it is not needed at the moment.

**Review by AtelyPham — COMMENTED** [review]
Left a comment.

**Comment by AtelyPham** [comment]
The CI is failing, @devpavan04!

File: apps/tangle-dapp/app/providers.tsx
Comment (vutuanlinh2k2) [inline apps/tangle-dapp/app/providers.tsx:1]: I already move the next theme provider [here](https://github.com/webb-tools/webb-dapp/blob/develop/libs/api-provider-environment/src/NextThemeProvider.tsx) to reuse it across dapps

File: apps/tangle-dapp/components/sideBar/sideBarActions.ts
Comment (vutuanlinh2k2) [inline apps/tangle-dapp/components/sideBar/sideBarActions.ts:1]: consider moving this into the **lib** folder

**Review by vutuanlinh2k2 — COMMENTED** [review]
Left some comments!

**Review by drewstone — APPROVED** [review]

File: libs/webb-ui-components/src/components/ChainChip/types.ts
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/ChainChip/types.ts:1]: Please revert this. Always use the library import `@webb-tools/*`. If you import from another library, avoid using relative paths.

File: libs/webb-ui-components/src/components/ChainChip/utils.ts
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/ChainChip/utils.ts:1]: Please revert this. Always use the library import `@webb-tools/*`. If you import from another library, avoid using relative paths.

File: libs/webb-ui-components/src/components/WalletConnectionCard/WalletConnectionCard.tsx
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/WalletConnectionCard/WalletConnectionCard.tsx:1]: Same here

File: libs/webb-ui-components/src/components/WalletConnectionCard/types.ts
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/WalletConnectionCard/types.ts:1]: Same here.

File: libs/webb-ui-components/src/components/buttons/types.ts
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/buttons/types.ts:1]: Same here.

File: libs/webb-ui-components/src/containers/ConfirmationCard/types.ts
Comment (AtelyPham) [inline libs/webb-ui-components/src/containers/ConfirmationCard/types.ts:1]: Same here.

**Review by AtelyPham — CHANGES_REQUESTED** [review]
Please revert some relative imports. Always use the library import @webb-tools/*. If you import from another library, avoid using relative paths.

**Review by AtelyPham — APPROVED** [review]

**Pull request merged**